picli/config.py

```diff
@@ -23,8 +23,8 @@ def _find_base_dir(self, config):
         configuration file. This will be the base directory
         that all other methods will assume.
         """
-        base_dir = os.path.dirname(os.path.dirname(config))
-        if not base_dir:
+        base_dir = os.path.normpath(os.path.join(os.path.abspath(config), '../..'))
+        if base_dir == os.getcwd():

```

Comment:
what if base_dir == ''?

Reply:
The idea here was basically to return the project directory which should sit two directories above the pi_global_vars.yml config file. This would then be passed down to the zip functions of the actions which would use this as the relative path inside the zip folder. 

There is probably a better way of handling this though..

Reply:
Removed this check as it was not required anymore